tpl/encoding/encoding_test.go

```diff
@@ -85,18 +85,21 @@ func TestJsonify(t *testing.T) {
 	ns := New()
 
 	for i, test := range []struct {
-		v      interface{}
-		expect interface{}
+		v       interface{}
+		spacing string
+		expect  interface{}
 	}{
-		{[]string{"a", "b"}, template.HTML(`["a","b"]`)},
-		{tstNoStringer{}, template.HTML("{}")},
-		{nil, template.HTML("null")},
+		{[]string{"a", "b"}, nil, template.HTML("[\n\"a\",\n\"b\"\n]")},
```

Comment:
This test should represent the current use case when a user does not specify the spacing **at all** .